sdk/src/main/java/com/google/cloud/dataflow/sdk/util/MergingActiveWindowSet.java

```diff
@@ -195,6 +191,7 @@ public void addActive(W window) {
 
   @Override
   public void remove(W window) {
+    Preconditions.checkState(isActive(window), "Window %s is not active", window);
```

Comment:
There are other calls to remove and I want to assert they are only removing active windows.